Robot Code/src/org/usfirst/frc/team3316/robot/commands/chassis/ExtendOmni.java

```diff
@@ -0,0 +1,89 @@
+package org.usfirst.frc.team3316.robot.commands.chassis;
+
+import org.usfirst.frc.team3316.robot.Robot;
+import org.usfirst.frc.team3316.robot.commands.DBugCommand;
+
+import edu.wpi.first.wpilibj.Timer;
+
+public class ExtendOmni extends DBugCommand
+{
+	private double timeout;
+	private double cancelTimeout;
+
+	private double timeoutStartTime;
+
+	private boolean leftSet, rightSet; // booleans for whether we've set already
+										// the pistons to open
+
+	public ExtendOmni()
+	{
+		requires(Robot.chassis); // We don't want the driver to stall the motors
+	}
+
+	protected void init()
```

Comment:
what happens if we call this when were already in omni deployed state?
Is there an easy way to know our state? (omni vs. pneuo)